projects/swift-protobuf/build.sh

```diff
@@ -16,21 +16,14 @@
 ################################################################################
 
 (
-cd llvm-project
+cd $SRC/llvm-project
 git apply ../llvmsymbol.diff
 cmake -G "Ninja" -DLIBCXX_ENABLE_SHARED=OFF -DLIBCXX_ENABLE_STATIC_ABI_LIBRARY=ON -DLIBCXXABI_ENABLE_SHARED=OFF -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD=X86 -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DLLVM_BUILD_TESTS=OFF -DLLVM_INCLUDE_TESTS=OFF llvm
 ninja -j$(nproc) llvm-symbolizer
 cp bin/llvm-symbolizer $OUT/
 )
 
 # build project
-mkdir swift-protobuf-fuzz
-cd swift-protobuf-fuzz
-swift package init --type=executable
-cp $SRC/fuzz_binary.swift Sources/swift-protobuf-fuzz/main.swift
-cp $SRC/Package.swift Package.swift
-cp ../swift-protobuf/Tests/SwiftProtobufTests/unittest.pb.swift Sources/swift-protobuf-fuzz/
-cp ../swift-protobuf/Tests/SwiftProtobufTests/unittest_import.pb.swift Sources/swift-protobuf-fuzz/
-cp ../swift-protobuf/Tests/SwiftProtobufTests/unittest_import_public.pb.swift Sources/swift-protobuf-fuzz/
+cd FuzzTesting
 swift build -c debug -Xswiftc -sanitize=address,fuzzer -Xswiftc -parse-as-library -Xswiftc -static-stdlib -Xswiftc -use-ld=/usr/bin/ld --static-swift-stdlib --sanitize=address
-cp .build/x86_64-unknown-linux-gnu/debug/swift-protobuf-fuzz $OUT/
+cp .build/x86_64-unknown-linux-gnu/debug/Fuzz* $OUT/ || true
```

Comment:
What does it need the `true` for?  When wouldn't the binaries have been built?

Reply:
> What does it need the true for? When wouldn't the binaries have been built?

The binaries are always built.
But `cp` returned an error because of `cp: omitting directory '.build/x86_64-unknown-linux-gnu/debug/FuzzBinary.build'`

Maybe a batter solution would be `swift package dump-package | jq '.targets[].name'`
What do you think ?

Reply:
Might be easier to just list the vs. getting extra complex.

You should also be able to shorten your paths to just `.build/debug/FuzzBinary`...

Reply:
The `jq` likely will find FuzzCommon so you'd have to deal with that one too.  ☹️

Reply:
> Might be easier to just list the vs. getting extra complex

I would like to have something generic for
1. other projects (which can copy/paste)
2. in case other targets are added in the future, you do not need to update oss-fuzz

So, I changed to `find -type f -name "Fuzz*" -executable`